src/PostalCodes/PostalCode.cs

```diff
@@ -342,7 +433,7 @@ public PostalCode ExpandPostalCodeAsLowestInRange()
                 }
             }
 
-            return CreatePostalCode(ToString(), _allowConvertToShort);
+            return this;
```

Comment:
Figured we don't need to recreate the same `PostalCode` object here if it's already in it's long form. Same as below for `ExpandPostalCodeAsHighestInRange`.